src/qml/editorwidgets/ExternalResource.qml

```diff
@@ -24,22 +24,23 @@ Item {
     fillMode: Image.PreserveAspectFit
 
     source: {
-      if (image.status === Image.Error)
+      if (image.status === Image.Error) {
         Style.getThemeIcon("ic_broken_image_black_24dp")
-      else if (currentValue)
-        if ( featureUseNativeCamera )
+      } else if (currentValue) {
+        if ( settings.value("useNativeCamera", false) )
```

Comment:
Is `settings` available here?

Reply:
Why not?

Reply:
But I guess it shoud be `valueBool` as well here.

Reply:
And it's not even needed, since the path is the same anyway...

Reply:
in this case, kill it here.
and it looks like it works, so all good :+1: